src/System.Runtime.WindowsRuntime/src/System/IO/IStorageFolderHandleAccess.cs

```diff
@@ -7,8 +7,8 @@
 
 namespace System.IO
 {
-    // Available in 14393 (RS1) and later
-    [ComImport]
+    // Available in 14393 (RS1) and later
```

Comment:
Is this space vs tab?

Reply:
No, actually.  It's three non-breaking spaces ('\u00A0') followed by one normal space ('\u0020').  I changed it to four normal spaces.